docker/firefox-browser-gui/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:14.04
```

Comment:
@mDibyo as discussed, it would be great if we could avoid some of the duplication in the Dockerfiles